### ISSUE TITLE
Lambda hard to read description

### ISSUE THREAD

**Issue by AMDphreak** [racket/racket#2872]
I am somewhat familiar with lambda functions, but this article is still pretty opaque to me. It would be helpful to have the examples grouped with their specific descriptions, rather than grouping all the examples together at the end of the description.

https://docs.racket-lang.org/reference/lambda.html#%28form._%28%28lib._racket%2Fprivate%2Fbase..rkt%29._lambda%29%29

**Comment by samth** [comment]
Is the guide section here https://docs.racket-lang.org/guide/lambda.html clearer? The intention is that the Guide provides an introduction, while the Reference is there to provide details and precision.

**Comment by AMDphreak** [comment]
> Is the guide section here https://docs.racket-lang.org/guide/lambda.html clearer? The intention is that the Guide provides an introduction, while the Reference is there to provide details and precision.

Not exactly. It is better, though.

It becomes more obvious what the fields are when I see the `((lambda (x y) (+ x y)) 1 2)` example, but the description above it is still murky. I come from a background in stereotypical college programming classes. I learned C++ at one college, and Java at another college. My first language was TI-BASIC, an imperative lang with no scope for variables, which I learned in middle and high school on a TI calculator without any docs.

Imperative language design is so burned into my thinking that every functional concept and term has to be spelled out for me. Declarative sentence structure works best in docs for anyone who has the same background as me.

I appreciate your help.
